pkg/controller/storagecluster/cephcluster.go

```diff
@@ -229,7 +233,8 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, nodeCount int, r
 	return cephCluster
 }
 
-func newExternalCephCluster(sc *ocsv1.StorageCluster, cephImage string) *cephv1.CephCluster {
+func newExternalCephCluster(sc *ocsv1.StorageCluster, cephImage string,
```

Comment:
Why did you add `reqLogger logr.Logger` if you are not using it?

Reply:
Yes. Usually if you don't have any error to report (or info to print), we don't require `reqLogger`.